internal/binarylog/sink.go

```diff
@@ -70,6 +70,7 @@ func (ws *writerSink) Write(e *pb.GrpcLogEntry) error {
 	b, err := proto.Marshal(e)
 	if err != nil {
 		grpclogLogger.Infof("binary logging: failed to marshal proto message: %v", err)
+		return err
```

Comment:
While you are here, could you also please change the `Infof` to `Errorf`. Thanks.

Reply:
Done